platform-ajax/src/main/java/com/softicar/platform/ajax/testing/selenium/engine/level/low/AjaxSeleniumLowLevelTestEngineViewport.java

```diff
@@ -34,7 +36,12 @@ public void setViewportSize(int width, int height) {
 		AjaxSeleniumTestSegment viewportSize = getViewportSize();
 		int extraWidth = windowSize.getWidth() - viewportSize.getWidth();
 		int extraHeight = windowSize.getHeight() - viewportSize.getHeight();
-		getWindow().setSize(new org.openqa.selenium.Dimension(width + extraWidth, height + extraHeight));
+
+		getWindow().setSize(new Dimension(width + extraWidth, height + extraHeight));
+
+		// wait for the window size to actually change
+		new WebDriverWait(webDriverSupplier.get(), Duration.ofSeconds(10))//
+			.until(drv -> !getViewportSize().equals(viewportSize));
```

Comment:
```suggestion
		new WebDriverWait(webDriverSupplier.get(), Duration.ofSeconds(10))//
			.until(driver -> !getViewportSize().equals(viewportSize));
```
not sure why we use `WebDriverWait` here and `FluentWait` everywhere else, what is the difference?